config/confignet/confignet.go

```diff
@@ -33,6 +37,12 @@ type AddrConfig struct {
 	DialerConfig DialerConfig `mapstructure:"dialer"`
 }
 
+func NewDefaultAddrConfig() AddrConfig {
```

Comment:
You have added the `New*` function for the `AddrConfig` type. This file also contains the `TCPAddrConfig` type. Can you add a `New*` function for that type as well?